packages/cli/src/platform/accelerate/enable.ts

```diff
@@ -61,7 +62,9 @@ export class Enable implements Command {
         throw new Error(payload.error.message)
       }
       return successMessage(
-        `Accelerate enabled. Use this generated API key in your Accelerate connection string to authenticate requests: ${payload.data.tenantAPIKey}`,
+        `Accelerate enabled. Use this generated API key in your Accelerate connection string to authenticate requests: \n${generateConnectionString(
+          payload.data.tenantAPIKey,
+        )}`,
```

Comment:
We could also add a link (we would need a shortlink) to https://www.prisma.io/docs/data-platform/accelerate/getting-started ?

Reply:
oh, to display users what are the next steps? that looks good ✨
I'll note this for the next iteration since we might want some product input here, what do you think?

Reply:
Sounds good!